File: CSharp/00. 사용방법.cs
/*
1. NET8.0 이상으로 프로젝트 생성
2. nuget 패키지 관리자
    install ConsoleTables
    install ebest.OpenAPI

3. 샘플폴더에 app_keys.cs 파일 생성
    app_keys.cs 파일에 아래와 같이 변수 세팅

    namespace CSharp;
    internal class Secret
    {
        public const string AppKey = "발급받은 앱Key";
        public const string AppSecretKey = "발급받은 앱 비밀Key";
    }


4. 샘플코드 실행

01 ~ 샘플코드는 로그인, 계좌조회, 시세 및 차트조회
10 ~ 조건검색, 실시간검색
20 ~ 웹소켓 을 이용한 실시간 시세
40 ~ 기타 샘플코드

* 샘플프로젝트는 이용된 TR에 해당되는 모델클래스를 nuget ebest.OpenAPI.Models를 이용하여 참조하였음
        - Program.cs 파일에 global using ebest.OpenAPI.Models; 추가)
        - ebest.OpenAPI.Models 에는 이베스트 전체 TR에 해당되는 클래스가 포함되어 있음
        - 일부만 필요할 경우, 필요한 TR에 해당되는 클래스만 개발 프로젝트에 복사해서 사용
*/

File: Python/00. 사용방법.py
'''
가능한 파이썬 64비트 최신버전 사용 권장 (3.11 이상)

패키지 설치 (가능한 최신버전 사용 권장)
pip install ebest

    일부 샘플은 prettytable, cryptography, padas, ta, matplotlib 패키지 필요
    pip install prettytable
    pip install cryptography
    pip install pandas
    pip install ta
    pip install matplotlib

    QT샘플은 PyQt6, qasync 필요
    pip install PyQt6
    pip install qasync


샘플 코드 이용
1. 샘플폴더에 app_keys.py 파일 생성
2. app_keys.py 파일에 아래와 같이 변수 세팅
    appkey = '발급받은 앱Key'
    appsecretkey = '발급받은 앱 비밀Key'
3. 샘플코드 실행

LS-OpenApi DevCenter 실행
python DevCenter.py
    LS증권 OpenApi 모든 요청/실시간 테스트 가능
    테스트베드 기능 포함
    개별 샘플파일 테스트 가능
    PyQt6, qasync, prettytable, cryptography 필요

샘플목록
01 ~ 로그인, 계좌조회, 시세 및 차트조회
10 ~ 조건검색, 실시간검색
20 ~ 웹소켓 을 이용한 실시간 시세, 해외선물 조회/실시간
30 ~ PyQt6를 이용한 GUI 샘플코드
40 ~ 기타 샘플코드

'''
